Client/Ecosia/UI/NTP/Impact/NTPImpactDividerFooter.swift

```diff
@@ -31,9 +31,7 @@ final class NTPImpactDividerFooter: UICollectionReusableView, ReusableCell, Noti
             dividerView.leadingAnchor.constraint(equalTo: leadingAnchor, constant: UX.dividerInset),
             dividerView.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -UX.dividerInset),
             dividerView.bottomAnchor.constraint(equalTo: bottomAnchor, constant: -UX.dividerBottom)
-        ])
-        
-        applyTheme()
```

Comment:
Combined with the suggestion above, I'd then keep it inside init as well so that the initial theme is applied